central/processlisteningonport/datastore/datastore_impl.go

```diff
@@ -129,7 +129,7 @@ func (ds *datastoreImpl) AddProcessListeningOnPort(
 		// and will not be stored.
 		if _, indicatorExists := indicatorsMap[indicatorID]; !indicatorExists {
 			countMetrics.IncrementOrphanedPLOPCounter(val.GetClusterId())
-			log.Warnf("Found no matching indicators for %+v", val)
+			log.Debugf("Found no matching indicators for %+v", val)
```

Comment:
Even in debug I don't think we can log the `process_args` as those could contain secrets.  We should find the subset of information that tells us something without anything sensitive.

Reply:
Done. `process_args` are no longer being logged here.

Reply:
I removed logging of `process_args` from everywhere.